Taskfile.yml

```diff
@@ -1278,3 +1277,13 @@ tasks:
     dir: v2/tools/mangle-test-json
     cmds:
       - go build
+
+  provision-oidc-storage-acct:
+    desc: Creates a storage account for use in OIDC federation with a random name in resource group "aso-wi-storage"
```

Comment:
Does this need a comment to explain this is for MSFT use only, because of security restrictions? Or maybe just to explain why it's needed in more generic terms?

Reply:
No we actually don't use this - it's for external users to make their own and use our tooling in their subs.

Our acct is static so doesn't need a cmd to create it dynamically.